alarms/Groupme/groupme_alarm.py

```diff
@@ -0,0 +1,89 @@
+#Setup Logging
+import logging
+log = logging.getLogger(__name__)
+
+#Local modules
+from ..alarm import Alarm
+from ..utils import *
+#Bot Implementation Requirements
+import requests
+from json import dumps as jsondumps
+
+
+class Groupme_Alarm(Alarm):
+  # notice: groupme does NOT support percent signs (%) in the message.
+  # I do not know how to fix this. Somebody fix this.
+  _defaults = {
+  'pokemon':{
+  'location_name':"<pkmn> until <24h_time>",
+      'text':"<pkmn> here until <24h_time> (<time_left>).\n IV: <atk>/<dfs>/<sta> \n Move 1: <move1> \n Move 2: <move2>"
+    },
+    'pokestop':{
+      'location_name':"Lured Pokestop",
+      'text':"A pokestop has been lured. Lure will expire at <24h_time> (<time_left>)."
+    },
+    'gym':{
+      'location_name':"Recently fallend gym",
+      'text':"A Team <old_team> gym has fallen and is now controlled by <new_team>"
+    }
+  }
+
+  #Gather settings and create alarm
+  def __init__(self, settings):
+    self.bot_id = settings.get('bot_id')
+    self.group_id = settings.get('group_id')
+    #Set Alerts
+    self.pokemon = self.set_alert(settings.get('pokemon', {}), self._defaults['pokemon'])
+    self.pokestop = self.set_alert(settings.get('pokestop', {}), self._defaults['pokestop'])
+    self.gym = self.set_alert(settings.get('gyms', {}), self._defaults['gym'])
+    log.info('Groupme Bot Sending')
+
+  #(Re)establishes Service connection
+  def connect(self):
+    log.info('Groupme bot connected (even though it doesnt really need to connect)')
+
+  #Set the appropriate settings for each alert
+  def set_alert(self, settings, default):
+    alert={
```

Comment:
Sorry I wasn't more clear - I mean it should be `alert = {`, not `alert={`. The name an stuff is fine.